lib/swagger/diff/diff.rb

```diff
@@ -112,24 +174,46 @@ def incompatible_response_attributes
         ret
       end
 
-      def incompatible_response_attributes_enumerator
+      def new_or_changed_response_attributes
```

Comment:
Since the request and response methods are nearly identical, you might consider extracting the diff method and passing it through.